scripts/pre_commit_linter.py

```diff
@@ -881,8 +883,84 @@ def _check_html_directive_name(all_files):
     return summary_messages
 
 
+def _check_directive_scope(all_files):
+    """This function checks that all directives have an explicit
+    scope key and it should not be scope: true.
+    """
+    print 'Starting directive scope check'
+    print '----------------------------------------'
+    summary_messages = []
+    files_to_check = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        and filename.endswith('.js')]
+    failed = False
+    directive_does_not_have_scope = True
```

Comment:
Use directive_has_isolate_scope or similar to avoid the double negative

Reply:
Done.